webpack/assets/javascripts/react_app/API.js

```diff
@@ -0,0 +1,11 @@
+import ServerActions from './actions/ServerActions';
+
+export default {
+  getStatisticsData(url) {
+    $.getJSON(url)
+      .success(
+        rawStatistics => ServerActions.receivedStatistics(rawStatistics)
+      )
+      .error(error => console.log(error));
```

Comment:
@tbrisker got above error for console statement although my .eslintrc sets no-console to warn. Thoughts?